src/styles/blocks/header.css

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.header__logo {
+  margin: 10px;
+  padding-right: auto;
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav {
+  display: flex;
+  flex-direction: row;
+  padding-right: 50px;
+}
+
+.header__nav__link {
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+  height: 15px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-right: 20px;
+  padding: 22.5px 0;
```

Comment:
Remove this padding, use `display: flex` with `align-items: center`

Usually, "half" or "broken numbers" in design are actually not fixed, but came from the centering or another type of alignment